examples/README.md

```diff
@@ -41,7 +57,7 @@ Please refer to gRPC Java's [README](../README.md) and
 [tutorial](https://grpc.io/docs/tutorials/basic/java.html) for more
 information.
 
-## Maven
+#### Maven
```

Comment:
Please use three instead of four. Ditto below.

Reply:
Done.